resources/code/cfn/windows-ad.template

```diff
@@ -0,0 +1,189 @@
+{
+  "AWSTemplateFormatVersion" : "2010-09-09",
+
+  "Description" : "This template creates a single server installation of Active Directory , Domain Name services, and creates the necessary scripts for AWS Lake Formation integration with Amazon EMR using ADFS as IdP. **WARNING** This template creates Amazon EC2 Windows instance and related resources. You will be billed for the AWS resources used if you create a stack from this template.",
+
+  "Parameters" : {
+
+    "SourceCidrForRDP" : {
+      "Description" : "IP Cidr from which you are likely to RDP into the instances. You can add rules later by modifying the created security groups e.g. 54.32.98.160/32",
+      "Type" : "String",
+      "MinLength" : "9",
+      "MaxLength" : "18",
+      "AllowedPattern" : "^([0-9]+\\.){3}[0-9]+\\/[0-9]+$"
+    },
+    "VPCPublicSubnet": {
+      "Type": "AWS::EC2::Subnet::Id",
+      "Description": "Public Subnet to be used for Windows AD Server, please choose LF-Workshop-PublicSubnetTwo",
+      "AllowedPattern" : ".+"

```

Comment:
If you are explicitly telling the audience to use LF-Workshop-PublicSubnetTwo, then no need to use the parameter and use the value directly inside CFN. Or you can give them options like either choose PB1 or PB2 through parameter